juniper_official/interface/check-ifl-state-in-out-traffic.rule

```diff
@@ -0,0 +1,425 @@
+                description "Sub Interfaces open-config sensor to collect telemetry data from network device";
+                open-config {
+                    sensor-name /interfaces/interface/subinterfaces/subinterface/state;
+                    frequency 60s;
+                }
+            }
+            /*
+             * Fields defined using sensor path. Map the longer sensor names
+             * to the shorter field names used in the rules.
+             */
+            field in-octets-stats-value {
+                sensor interfaces-oc {
+                    path "/interfaces/interface/subinterfaces/subinterface/state/counters/{{in-stats-name}}";
+                    zero-suppression;
+                }
+                type integer;
+                description "Sub Interface statistics counter (in-octets) value.";
+            }
+            field out-octets-stats-value {
+                sensor interfaces-oc {
+                    path "/interfaces/interface/subinterfaces/subinterface/state/counters/{{out-stats-name}}";
+                    zero-suppression;
+                }
+                type integer;
+                description "Sub Interface statistics counter (out-octets) value.";
+            }			
+            field interface-name {
+                sensor interfaces-oc {
+                    where "/interfaces/interface/@name =~ /{{ifd-name}}/";
+                    path "/interfaces/interface/@name";
+                }
+                type string;
+                description "Interfaces to be monitored";
+            }
+            field sub-interface-index {
+                sensor interfaces-oc {
+                    where "/interfaces/interface/subinterfaces/subinterface/@index =~ /{{ifl-no}}/";
+                    path "/interfaces/interface/subinterfaces/subinterface/@index";
+                }
+                type string;
+                description "Interfaces to be monitored";
+            }
+            field ifl-admin-status {
+                sensor interfaces-oc {
+                    path /interfaces/interface/subinterfaces/subinterface/state/admin-status;
+                }
+                type string;
+                description "Subinterface admin status";
+            }
+            field ifl-oper-status {
+                sensor interfaces-oc {
+                    path /interfaces/interface/subinterfaces/subinterface/state/oper-status;
+                }
+                type string;
+                description "Subinterface operational status";
+            }			
+            /*
+             * Fields defined with variables and constant values
+             */
+            field high-threshold {
+                constant {
+                    value "{{octets-high-threshold}}";
+                }
+                type integer;
+                description "Traffic high threshold.";
+            }
+            field low-threshold {
```

Comment:
threshold should be in mbps like in ifd rule

Reply:
added field for  mbps and changed variable name